sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -4,7 +4,6 @@
     <AssemblyTitle>Digital Twins SDK</AssemblyTitle>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
 
-    <EnableApiCompat>false</EnableApiCompat>
```

Comment:
Is removal alone sufficient? We haven't done the post-release work yet, but it is on our task list.

Reply:
Yes with my update to how ApiCompat works you won't get Api checks done until you add an ApiCompatVersion in your project. That will happen automatically as part of our version bump PR once you ship a GA.

Reply:
Do we have to ship a GA, or can our shipped public preview work for getting this task done?

Reply:
You can add any shipped version if you want to start a baseline to have the API's verified againt it.

Reply:
I wonder if there really is a use case where you would want to specify some version other than what the version ApiCompatVersion gets auto-updated to, or is storing off the version number in the project just a technical necessity? I guess what I'm getting at is would it be possible to have this value not be configurable? Turning off or on could be configurable but selecting a version would not be. I suppose this might not be possible since we would need to store off the version to do the compat check separately from the actual package version to account for previews, but my worry is that this might get manually updated to an incorrect value. If it was in a shared props file outside of the package, it might make this less likely and also avoid potential confusion around what the value should be set to.

Reply:
As you point out we would have to keep it somewhere and personally I think keeping it right next the current version is the best spot for it. Sure someone could change it but they could change it no matter were we put it. The good news is that our automation for bumping the version will always try to update it once we ship a GA so it will eventually get reset back.

As for specifically setting it to something else I can imagine cases like digitaltwins that wants to lock the api surface while still in preview and so they could set it to a earlier preview version and then ensure no breaking changes happen after that.

Reply:
I think the most likely situation where someone might incorrectly change this property would be when doing a Preview version for a GA library, as this is the only time we need to manually change the version. Since the property is right next to the package version, they might think that they need to update both. However, I think the build would fail if they do this since that version hasn't been shipped yet, so I would agree that the potential for harm is pretty low. BTW this is awesome that we no longer have to remember to update this when shipping a new GA version.